locale/de/foreman_discovery.po

```diff
@@ -415,10 +446,61 @@ msgstr "Anzahl der Einträge pro Anfrage"
 msgid "paginate results"
 msgstr "Ergebnisse nummerieren"
 
-msgid ""
-"required if value is not inherited from host group or default password in "
-"settings"
+msgid "puts the rules in order, low numbers go first. Must be greater then zero"
+msgstr ""
+
+msgid "query to match discovered hosts for the particular rule"
+msgstr ""
+
+msgid "represents rule name shown to the users"
+msgstr ""
+
+msgid "required if value is not inherited from host group or default password in settings"
 msgstr "erforderlich, wenn der Wert nicht von einer Hostgruppe oder einem Standardpasswort in den Einstellungen vererbt ist"
 
 msgid "sort results"
 msgstr "Ergebnisse sortieren"
+
+msgid "the hostgroup that is used to auto provision a host"
+msgstr ""
+
+#~ msgid "Create a discovered host"
+#~ msgstr "Einen entdeckten Host erstellen"
+
+#~ msgid "Discovered Host Pool"
+#~ msgstr "Entdeckter Host-Pool"
+
+#~ msgid "Discovery rules"
+#~ msgstr "Entdeckungsregeln"
+
+#~ msgid "Discovery widget"
+#~ msgstr "Entdeckungs-Widget"
+
+#~ msgid "Failed to import facts for discovered host"
+#~ msgstr "Importieren von Fakten für entdeckten Host fehlgeschlagen"
+
+#~ msgid "Failed to import facts for discovered host: %s"
+#~ msgstr "Importieren von Fakten für entdeckten Host fehlgeschlagen: %s"
+
+#~ msgid "Imported discovered host"
+#~ msgstr "Entdeckter Host importiert"
+
+#~ msgid "Invalid facts: hash does not contain IP address"
+#~ msgstr "Ungültige Fakten: Hash enthält keine IP-Adresse"
+
+#~ msgid "Invalid facts: hash does not contain the required fact '%s'"
+#~ msgstr "Ungültige Fakten: Der Hash enthält den benötigten Fakt nicht: %s"
+
+#~ msgid "Show fact as an extra column in the discovered hosts list"
+#~ msgstr "Fakt als zusätzliche Spalte in der Liste entdeckter Hosts anzeigen"
+
+#~ msgid "The default fact name to use for the MAC of the system"
+#~ msgstr "Der Standard-Faktenname zum Verwenden für die MAC des Systems"
+
+#~ msgid "Total pool size"
+#~ msgid_plural "Total pool size"
+#~ msgstr[0] "Gesamt-Pool-Größe"
+#~ msgstr[1] "Gesamt-Pool-Größe"
+
+#~ msgid "hash containing facts for the host"
+#~ msgstr "Hash, der Fakten für den Host enthält"
```

Comment:
Was not sure how to disable those "backups". I assume these gets overwritten after we pull from Transifex...